<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInitWidget</class>
 <widget class="QWidget" name="CInitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CInitWidget</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="skgui.qrc">:/images/project_logo</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>376</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="skgui.qrc">:/images/project_name</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_load">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>启动中......</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>23</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnClose{
  padding: -1;
  border-image:url(:/images/btn_close_normal);
}

QPushButton#btnClose:hover {
  padding: -1;
  border-image:url(:/images/btn_close_hover);
}

QPushButton#btnClose:pressed {
  padding: -1;
  border-image:url(:/images/btn_close_down);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="skgui.qrc"/>
 </resources>
 <connections/>
</ui>
